pom.xml

```diff
@@ -55,6 +55,7 @@
     <modules>
         <module>timachine-core</module>
         <module>timachine-maven-plugin</module>
+        <module>timachine-maven-plugin-test</module>
```

Comment:
Reasonable, but I think we should skip the deployment of this module. (It has no meaning to deploy)

Could you add 

```
<plugin>
  <groupId>org.apache.maven.plugins</groupId>
  <artifactId>maven-deploy-plugin</artifactId>
  <version>X.Y</version>
  <configuration>
    <skip>true</skip>
  </configuration>
</plugin>
```

Then I'll merge this PR, thanks a lot!